config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,28 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+if Rails.env.test? || Rails.env.development?
+  eu_bank = EuCentralBank.new
```

Comment:
You should also `update_rates` the same way you're doing for the new CurrencyLayer bank below

Reply:
Oh yes, done.